### DISCUSSION TITLE
Add compute shader？

### DISCUSSION THREAD

**Discussion by swq0553** [google/filament#4554] [Ideas]
In filament document，there is a piece of code about compute shader code here：[https://google.github.io/filament/Filament.html#annex/lightassignmentwithfroxels](url)
Currently，filament does not support compute shader.
Is it possible to add support for compute shader in the filament ,so one can do some compute-related work .

**Comment by romainguy** [comment]
We will once we can drop support for OpenGL ES 3.0.
